tests/unit/components/materialize-radios-test.js

```diff
@@ -0,0 +1,33 @@
+import {
+  moduleForComponent,
+  test
+} from 'ember-qunit';
+
+
+import {
+  disabledGroupTest,
+  groupItemsRenderTest,
+  initialSelectionTest
+} from '../../helpers/selectable-item';
+
+
+moduleForComponent('materialize-radios', {
+  // Specify the other units that are required for this test
+  // needs: ['component:foo', 'helper:bar']
+});
+
+test('it renders', function(assert) {
+  assert.expect(2);
+
+  // Creates the component instance
+  var component = this.subject();
+  assert.equal(component._state, 'preRender');
+
+  // Renders the component to the page
+  this.render();
+  assert.equal(component._state, 'inDOM');
+});
+
+// disabledGroupTest();
```

Comment:
Added. I have made a design decision to disallow `{{materialize-radio}}` outside the context of a `{{materialize-radios}}` group, so it is not tested independently. In the event that someone tries this, they'll get an error message telling them that it won't work